phoenix-core/src/main/java/org/apache/phoenix/util/ViewUtil.java

```diff
@@ -629,4 +631,37 @@ public static void addTagsToPutsForViewAlteredProperties(List<Mutation> tableMet
 
         }
     }
+
+    public static ParseNode getViewWhereWithViewTTL(StatementContext context, PTable table)
```

Comment:
Let's add a unit test for this new method. It's fine for ViewUtilTest to just contain a test for this new method for now.